backend/src/api/query.rs

```diff
@@ -47,6 +47,11 @@ impl Query {
         Event::load_by_id(id, context).await
     }
 
+    /// Returns a list of all events the current user has read access to
+    async fn events(context: &Context) -> ApiResult<Vec<Event>> {
+        Event::load(context).await
+    }
```

Comment:
We certainly need a better solution for this. That will be way too many events. The SQL query takes 700ms, the graphql response is 680KB (only 176 compressed) and chosing a video from the select is bad UX. As a first version, sure, this is fine. Just wanted to note it here.

Reply:
Sure, I'm aware. 🤷‍♀️